common-rest/src/main/java/org/uniprot/api/rest/openapi/APIDocConfig.java

```diff
@@ -119,6 +122,51 @@ private void customiseSchema(OpenAPI openAPI) {
         cleanStringSerialiserSchemaObjects(schemaMap);
     }
 
+    private static void customiseSearchResultEntity(
+            OpenAPI openAPI, Map<String, Schema> schemaMap) {
+        List<String> searchEntries =
+                openAPI.getPaths().values().stream()
+                        .map(PathItem::getGet)
+                        .filter(Objects::nonNull)
+                        .map(Operation::getResponses)
+                        .map(ApiResponses::getDefault)
+                        .map(ApiResponse::getDescription)
+                        .filter(s -> !s.equals("default response"))
+                        .collect(Collectors.toList());
+        if (!searchEntries.isEmpty()) {
```

Comment:
I'm a bit unclear on what you are trying to do here?

Reply:
I refactored this code to improve logic.